CHANGELOG.md

```diff
@@ -8,8 +8,8 @@ Subheadings to categorize changes are `added, changed, deprecated, removed, fixe
 
 # Changelog
 
-The latest published Vello release is [0.5.0](#050---2025-05-08) which was released on 2025-05-08.
-You can find its changes [documented below](#050---2025-05-08).
+The latest published Vello release is [0.5.1](#051---2025-08-21) which was released on 2025-08-21.
+You can find its changes [documented below](#051---2025-08-21).
```

Comment:
These dates are now wrong! (This is a reason to make the changelog update in the same PR as bumping the version number)